vorestation.dme

```diff
@@ -1617,10 +1619,10 @@
 #include "code\modules\ai\ai_holder_targeting.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai_vr.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai_vr.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai_vr.dm"

```

Comment:
Are you a goose?

Reply:
No, oddly enough Dreammaker did that automatically. I have no idea why.